src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -248,7 +248,6 @@ public TypeSymbolWithAnnotations SetIsAnnotated(CSharpCompilation compilation)
                 }
                 else
                 {
-                    // PROTOTYPE(NullableReferenceTypes): what if the Nullable<T> type is missing? (are we missing diagnostics?)
```

Comment:
>// PROTOTYPE(NullableReferenceTypes): what if the Nullable<T> type is missing? (are we missing diagnostics?) [](start = 20, length = 108)

I don't think simply removing this comment is the right thing to do because use-site errors are not reported here, and after the change in BindNamespaceOrTypeOrAliasSymbol they won't be reported there either.